### ISSUE TITLE
Sound not muted on PC while streaming

### ISSUE THREAD

**Issue by matatom** [acidhax/chromecast-audio-stream#63]
Sound is not muted on PC while streaming to Chromecast. Mixer is changing the volume of my laptop speakers and the stream volume parallel, resulting sound coming from Laptop speakers and chromecast connected speakers too. 
Maybe I miss something and i'm not using it the proper way, but it is definitely an issue for me.

**Comment by Funcrusha** [comment]
Yeah, I have the same issue along with the sound delay mentioned in thread #64. Win10 64, using an access point on my Android phone (LTE) - maybe that's the source of the lag?

**Comment by ghost** [comment]
I am also having the same issue as stated above:

> Sound is not muted on PC while streaming to Chromecast. Mixer is changing the volume of my laptop 
> speakers and the stream volume parallel, resulting sound coming from Laptop speakers and chromecast connected speakers too. 

Any update on why this is happening (or better yet a fix!)

Thanks!

**Comment by datgreekguy** [comment]
I found a "fix" - win10 64, streaming off my laptop to TV 

1) open volume mixer
2) unmute everything. 
3) mute every app except the source of the audio (in my case, media player) and "ffmpeg"
4) put those two channels at 100% volume
5) mute "speakers" or whatever the device audio channel is named. 

Getting audio going to tv, nothing out of my laptop. Seems to always work. 

I hope this helps anyone in any way.

**Comment by shoummashams** [comment]
Having this exact same issue, along with audio lag on Win 10 64bit.

**Comment by wysocki** [comment]
PLEASE DON'T FIX THIS! I"ve been trying to find a way of having my local pc feed my main amps with audio while at the same time, casting to my remote devices - Finally I found it! I guess it would be a  good thing to make selectable feature tho.  But I agree with the earlier post that a workaround is to play with the mixer mutes.

**Comment by kataggart** [comment]
Has anybody resolved this yet? Super annoying.
